public/js/utils/sources-tree.js

```diff
@@ -102,7 +103,7 @@ function getURL(source: TmpSource): { path: string, group: string } {
 function addToTree(tree: any, source: TmpSource) {
   const url = getURL(source);
 
-  if (IGNORED_URLS.includes(url) ||
+  if (includes(IGNORED_URLS, url) ||
```

Comment:
Switching it to `indexOf` as this is a small nit.

I can see the perspective of minimizing lodash's API to helpful utility methods. With that said, I also think lodash is helpful smoothing over support for native methods while vendors catch up. Using lodash includes is fairly safe as we could safely replace it with native support when we feel comfortable bumping Node support to v6.